packages/office-addin-cli/src/commands.ts

```diff
@@ -4,15 +4,15 @@
 import { logErrorMessage } from "./log";
 import { usageDataObject } from "./defaults";
 
-export async function upgrade(manifestPath: string) {
+export async function convert(manifestPath: string) {
   try {
     // Convert manifest .xml to manifest.json
     // Do any more needed modifications on the code?
```

Comment:
I know we don't have the xml -> json conversion library yet, but you should consider the other things that should be done and start adding the code for those.  
One example would be verifying the package.json has the needed package versions for the json manipulation and publish command (even if you don't have the final package versions, you can write the code to check for the current versions so that we just have to update the version numbers in the future).